File: test/py/case96_class_static.py
# このファイルは `test/py/case96_class_static.py` のテスト/実装コードです。
# 役割が分かりやすいように、読み手向けの説明コメントを付与しています。
# 変更時は、既存仕様との整合性とテスト結果を必ず確認してください。

class Counter96:
    total: int = 0

    def add(self, x: int) -> int:
        self.total = self.total + x
        return self.total


if __name__ == "__main__":
    c: Counter96 = Counter96()
    print(c.add(5))

File: test/cs/case96_class_static.cs
// このファイルは `test/cs/case96_class_static.cs` の変換結果（C#サンプル）です。
// Python入力との対応関係を追いやすくするため、日本語コメントを付与しています。
// 仕様変更時は、対応する Python 側ケースと合わせて更新してください。

using System;

public static class Program
{
    public class Counter96
    {
        public static int total = 0;

        public int add(int x)
        {
            Counter96.total = (Counter96.total + x);
            return Counter96.total;
        }
    }

    public static void Main(string[] args)
    {
        Counter96 c = new Counter96();
        Console.WriteLine(c.add(5));
    }
}
